homeassistant/components/myuplink/sensor.py

```diff
@@ -102,4 +157,36 @@ def __init__(
     def native_value(self) -> StateType:
         """Sensor state value."""
         device_point = self.coordinator.data.points[self.device_id][self.point_id]
+        if device_point.value == -32768:
```

Comment:
What does this check mean?

Reply:
Normally the API will not include device_point that are permanently unavailable. The API will return -32768 if data is temporarily unavailable. I.e. "Unknown" with HA terminolgy. It happens on two data_points in my F730. I think it is better to show "Unknown" than showing -32768.

Reply:
That constant kind of needs to be defined in the library, but it at least need a constant defined for it here.